community.md

```diff
@@ -1,12 +0,0 @@
----
-title: Connect with us
```

Comment:
@lilin90 @CaitinChen PTAL whether we can remove this file in the docs repo. Thanks!

Reply:
@TomShawn Yes. I think we can remove it.

Reply:
Get. Thanks!